ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/OptimizeService.java

```diff
@@ -173,14 +173,14 @@ private void clearTableCache(TableIdentifier tableIdentifier) {
     TableOptimizeItem tableItem = cachedTables.remove(tableIdentifier);
     optimizeQueueService.release(tableIdentifier);
     try {
-      deleteTableOptimizeRuntime(tableIdentifier);
+      tableItem.optimizeTasksClear(false);
     } catch (Throwable t) {
-      LOG.debug("failed to delete  " + tableIdentifier + " runtime, ignore", t);
+      LOG.debug("failed to delete " + tableIdentifier + " optimize task, ignore", t);
     }
     try {
-      tableItem.clearOptimizeTasks();
```

Comment:
There are two similar methods in `TableOptimizeItem`:
- clearOptimizeTasks
- optimizeTasksClear

I think we should remove one of them.

Reply:
OK